.github/ISSUE_TEMPLATE/downloads.yml

```
@@ -0,0 +1,44 @@
+---
+name: Downloads on [git-scm](https://git-scm.com/) not working
```

I don't think the title here is interpreted as markdown; the user will see the extra punctuation here. Probably we want just:

```suggestion
name: Downloads on https://git-scm.com/ not working
```